src/processors/other/Tremolo.cpp

```diff
@@ -118,29 +121,51 @@ void Tremolo::processAudio (AudioBuffer<float>& buffer)
 {
     const auto numSamples = buffer.getNumSamples();
     waveBuffer.setSize (1, numSamples, false, false, true);
+    modBuffer.setSize (1, numSamples, false, false, true);
     phaseSmooth.setTargetValue (*rateParam * MathConstants<float>::pi / fs);
     waveSmooth.setTargetValue (*waveParam);
 
-    // fill wave buffer (-1, 1)
-    dsp::AudioBlock<float> waveBlock { waveBuffer };
-    dsp::ProcessContextReplacing<float> waveCtx { waveBlock };
-    fillWaveBuffer (waveBlock.getChannelPointer (0), numSamples, phase);
+    if (inputsConnected.contains (ModulationInput))
+    {
+        // get modulation buffer from input (-1, 1)
+        modBuffer.copyFrom (0, 0, getInputBuffer (ModulationInput), 0, 0, numSamples);
+        makeBufferMono(modBuffer);
```

Comment:
So I'm not sure this logic is exactly right... the way it's written currently, still only the first channel of the input buffer is being used. Really what we want is probably something like:
```cpp
const auto& modInputBuffer = getInputBuffer (ModulationInput);
modBuffer.copyFrom (0, 0, modInputBuffer, 0, 0, numSamples);

if (const auto modInputNumChannels = modInputBuffer.getNumChannels(); modInputNumChannels > 1)
{
    for (int ch = 1; ch < modInputNumChannels; ++ch)
        modBuffer.addFrom (0, 0, modInputBuffer, ch, 0, numSamples);

    modBuffer.applyGain (1.0f / (float) modInputNumChannels);
}
```

Reply:
Ah I see, I was only copying the 1st channel over before and calling `makeBufferMono`() was redundant. Thanks for catching that